inference-engine/tests/functional/inference_engine/ir_serialization/rt_info_deserialization.cpp

```diff
@@ -238,6 +240,9 @@ TEST(RTInfoDeserialization, NodeV11) {
             </output>
         </layer>
         <layer name="output" type="Result" id="2" version="opset8">
+            <rt_info>
+                <attribute name="old_api_map" version="0" order="0,3,1,2" element_type="undefined"/>
+            </rt_info>
```

Comment:
BTW, should not it be a part of tensor, not an operation attribute? different tensor may have different old api map in a theory

Reply:
This implementation of OldAPI map attribute in nGraph doesn't forbit it to be part of tensor. As it is a part of rt_info it can be placed <port> section the same way as other rt_info like in InputAndOutput* tests.
I didn't test this particular case as in MO implementation old_api_map for Parameter and Result is always part of an operation.

Reply:
@jane-intel is it needed to move old_api_map attribute to `<port>` section in MO implementation?